_posts/2020-01-13-ols1-announcement.md

```diff
@@ -27,20 +27,20 @@ We welcome our mentors, {{ p-mentors | join: ', ' }}. We are extremely grateful
 
 ### The Program
 
-We will begin our program this week with a mentoring training call, and mentor-mentee introductions. Check out the complete schedule and plans for OLS-1 here: [{{ site.url }}/ols-1](/ols-1).
+We will begin our program this week with a mentoring training call, and mentor-mentee introductions. Check out the complete schedule and plans for OLS-1 here: [{{ site.url }}/ols-1]({% link openseeds/ols-1/index.md %}).
 
 You can keep track of our program, the progress of our first cohort and future announcements by following our twitter profile [@{{ site.twitter }}](https://twitter.com/{{ site.twitter }}) or joining our [community Gitter channel](https://gitter.im/{{ site.gitter }}).
 
 We invite new contributions to the program as a [new issue on the GitHub repo]({{ site.github.repository_url }}/issues) or by [email to the team](mailto:{{ site.email }}).
 
 Once again, let's welcome our mentors and mentees to this program!
 
-## Project details ([click here for full description](/ols-1/projects-participants/))
+## Project details ([click here for full description](/openseeds/{{ cohort }}/projects-participants))
```

Comment:
`cohort` is not assigned here

Reply:
Actually, I'm just having a good look at this. 👀 `cohort` is assigned in all _ols-x announcement_ blog posts. Recall that these posts are dynamically generated. 😀

Reply:
👀 I'm actually looking at this properly now and recalling that all _ols-x announcement_ blog posts are generated dynamically. That means, `cohort` is assigned in all such pages.

But the variable seems to not work when used in this line. 🤔